src/applications/personalization/dashboard-2/components/NotificationCTA.jsx

```diff
@@ -5,7 +5,7 @@ const NotificationCTA = ({ CTA }) => {
 
   return (
     <a
-      aria-label={`${ariaLabel} (opens in new tab)`}
+      aria-label={ariaLabel ? `${ariaLabel} (opens in new tab)` : ''}
```

Comment:
I felt like the CTA I made didn't need an aria label since its label was clear enough. If we keep using this component we'll also need to make opening a new tab optional.